src/lib/defaultCommands.js

```diff
@@ -596,6 +652,18 @@ define(function (require) {
 				});
 
 				editor.createDropDown(caller, 'insertlink', content);
+
+				// Check it is an email address.
+				if ($anchor.attr('href') &&
+					$anchor.attr('href').substr(0, 7) !== 'mailto:') {
+				// Get details of highlighted and put in drop down.
+					if ($anchor.attr('href')) {
```

Comment:
Oops, that shouldn't be there as its previously checked my bad.. Lol. Its also present on the mail one.. I think the reason i did this is because I had to check one isn;t an email so it didn't grab the mailto string or remove that some chars from the link if link was selected.

Also we should replace 
`$anchor.attr('href').substr(0, 7) !== 'mailto:')`
With
`$anchor.attr('href').indexOf('mailto:') < 0;`
Its shorter and faster (Slightly/maybe lol)! sorry for not putting it in the first place.

Thanks for the corrections!